tests/base.py

```diff
@@ -27,30 +27,102 @@ def setUp(self):
         self.app_test = app.test_client()
         with app.app_context():
             Base.metadata.create_all(bind=engine)
-
             command.stamp(self.alembic_configuration, 'head')
-            command.downgrade(self.alembic_configuration, '-1')
+            command.downgrade(self.alembic_configuration, 'base')
             command.upgrade(self.alembic_configuration, 'head')
 
+            self.bouquet_201 = {
```

Comment:
How come you are getting authentication details from the environment variables yet we are now storing them in the db?

Reply:
For testing purposes